lib/datastore/transaction.js

```diff
@@ -84,16 +89,11 @@ nodeutil.inherits(Transaction, DatastoreRequest);
  * @param {function} callback - The function to execute within the context of
  *     a transaction.
  *
+ * @private
```

Comment:
Privatize `transaction#begin` without changing the name yet. Should I go ahead and change it to `begin_` to follow convention?